nuxt.config.ts

```diff
@@ -23,7 +23,10 @@ export default defineNuxtConfig({
     planshipApiClientSecret: process.env.PLANSHIP_API_CLIENT_SECRET,
 
     public: {
+      planshipApiClientUrl: process.env.PLANSHIP_API_CLIENT_URL,
```

Comment:
These should be documented in the README and/or have useful defaults (prod).

Reply:
> These should be documented in the README and/or have useful defaults (prod).

The SDK defaults to prod (`api.planship.io`) if no values are specified. No need to document since these options are moot outside of our dev/staging environment.

Reply:
I'm just going to remove these completely for now, and will do the same in the https://github.com/planship/planship-nextjs-example